src/main/java/com/marksem/crm/entity/Booking.java

```diff
@@ -0,0 +1,31 @@
+package com.marksem.crm.entity;
+
+import lombok.*;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Table(name = "booking")
+@Data
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
```

Comment:
You need this only in `BaseEntity.java`. You can delete it for the rest of entities that do not contain any JPA auditing annotations